lib/wsdl.js

```diff
@@ -1558,7 +1558,11 @@ WSDL.prototype.objectToXML = function(obj, name, namespace, xmlns, first, xmlnsA
       }
 
       if (!Array.isArray(child)) {
-        parts.push(['<', ns, name, attr, xmlnsAttrib, '>'].join(''));
+        var insNull = '';
+        if(typeof child === 'object' && child === null) {
```

Comment:
why do we need an extra check for `typeof child === 'object'`? wouldn't it be sufficient to check for `null` or `undefined` only. or, better to say, wouldn't it be safer (as we could also detect an empty String) to check for the "falsyness" of it by doing `if (!child) {...`?